client/components/header-cake/back.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ObserveWindowSizeMixin from 'lib/mixins/observe-window-resize';
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+
+	displayName: 'HeaderCakeBack',
+
+	mixins: [ ObserveWindowSizeMixin ],
+
+	propTypes: {
+		onClick: PropTypes.func,
```

Comment:
In this particular case it is actually optional because of the spacer usage. In order to give the title the most about of space possible and still keep in centered, the corners were changed to display the same text in both places (the right one is hidden). The right corner still uses the back component, but doesn't include any `onClick` props.